_docs/runtime/installation.md

```diff
@@ -1,10 +1,91 @@
 ---
-title: "Installation and configuration"
+title: "Runtime installation and configuration"
 description: ""
 group: runtime
 toc: true
 ---
 
-Coming soon
 
-Documentation on how to install it, what are the prerequisites for installing it and what are the different options for various Cloud providers
+Codefresh runtime installs Codefresh Software Development Platoform (CSDP), comprising Argo CD ecosystem components and Codefresh-specific components. The Argo CD components are derieved from a conformed fork.
+
+There are two parts to installing runtimes:
+1. Installing the Codefresh CLI, a one-time action, typically required only for initial CDSP setup.
+2. Installing the Codefresh runtime from the CLI. You can install and provision as many runtimes as you need to. Every runtime is installed at the cluster-level.
```

Comment:
it's installed in a specific namepsace and only once on a cluster for now.
we will allow multi installations on same cluster on multi namespaces in near future.

so not sure we should go into this for now

Reply:
or keep it like it is for now till we will support it

Reply:
I changed as needed for now. Will update when we support multi-namespace installs